packages/app/src/Element/Text.css

```diff
@@ -3,6 +3,8 @@
   line-height: 24px;
   white-space: pre-wrap;
   word-break: break-word;
+  margin-top: 12px;
```

Comment:
Not sure if this is the best solution, where does the extra padding come from, can we just make sure to use `overflow: hidden` on those elements because this will just add padding thats not needed for non-markdown

Reply:
Another issue.

![image](https://user-images.githubusercontent.com/10111/232441094-438d37fd-ca07-41c9-a056-8671bfff4c0c.png)

I can't read this Chinese text (because I'm Japanese), but this is difficult to read for them too.

Reply:
We probably need to adjust the heading element line-height here, i think it was raised before as an issue